src/data.rs

```diff
@@ -0,0 +1,398 @@
+use anyhow::anyhow;
+use chrono::{offset::TimeZone, DateTime, Datelike, NaiveTime};
+use chrono_tz::Europe::Kiev;
```

Comment:
Oh no! Kiev instead of Kyiv 😡

Reply:
XD

I don't make 'em rules, it's the library, and it itself depends on a public database] Best I can do is add `as Kyiv`

Reply:
Will do]

Reply:
> XD
> 
> I don't make 'em rules, it's the library, and it itself depends on a public database] Best I can do is add `as Kyiv`

Well, you can always open a pull request :)

Reply:
> > XD
> > I don't make 'em rules, it's the library, and it itself depends on a public database] Best I can do is add `as Kyiv`
> 
> Well, you can always open a pull request :)

Ok, they are maybe going to rename our capital

![image](https://github.com/WizardPlatypus/velik-schedule-bot/assets/70100129/ab865d41-d2b9-442a-9540-7c09e9c3eac6)

Reply:
Nice of you to check! Thx